Makefile

```diff
@@ -420,7 +420,7 @@ $(LIBWASI_EMULATED_PROCESS_CLOCKS_OBJS): CFLAGS += \
     -I$(LIBC_BOTTOM_HALF_CLOUDLIBC_SRC)
 
 include_dirs:
-	$(RM) -r "$(SYSROOT)"
+	$(RM) -r "$(SYSROOT_INC)"
```

Comment:
I think maybe we should completely remove this line?  If we are separating the cleaning from the building?

Reply:
OK

Reply:
I thought it would break `make; make`, but it doesn't!